src/app/metadata/features/dataset-expansion-panel/dataset-expansion-panel.component.html

```diff
@@ -1,95 +1,183 @@
+              <p>
+                <strong>Accession ID: </strong>
+              </p>
+              <p>
+                {{
+                  studiesSummary().count > 0 ? studiesSummaryStats().accession : 'N/A'
+                }}
+              </p>
+            </div>
+          </div>
+          <div class="flex">
+            <div class="me-3 pt-4 text-5xl text-neutral-700">
+              <mat-icon inline>genetics</mat-icon>
+            </div>
+            <div>
+              <p><strong>Sample info:</strong></p>
+              <p>
+                <strong>{{ samplesSummary().count }}</strong> Sample{{
+                  samplesSummary().count === 1 ? '' : 's'
+                }}
+                @if (samplesSex() !== []) {
+                  (Sex:
+                  @for (sex of samplesSex(); track sex.value; let last = $last) {
+                    {{ sex.count }} {{ sex.value }}{{ last ? ')' : ', ' }}
+                  }
+                }
+              </p>
+              <p>
+                <strong>{{ samplesTissues().length }}</strong>
+                Tissue{{ samplesTissues().length === 1 ? '' : 's' }}
+              </p>
+              @if (samplesTissues().length > 0) {
+                <ul class="ms-6">
+                  @for (tissue of samplesTissues(); track tissue.value) {
+                    <li>
+                      <mat-chip class="small-chips mb-0.5 text-xs"
+                        ><span class="uppercase">{{ tissue.value }} </span>
+                        <strong>{{ tissue.count }}</strong></mat-chip
+                      >
+                    </li>
+                  }
+                </ul>
+              }
+              <p>
+                <strong>{{ samplesPhenotypes().length }}</strong>
+                Phenotypic Feature{{ samplesPhenotypes().length === 1 ? '' : 's' }}
+              </p>
+              @if (samplesPhenotypes().length > 0) {
+                <ul class="ms-6">
+                  @for (
+                    phenotypic_feature of samplesPhenotypes();
+                    track phenotypic_feature.value
+                  ) {
+                    <li>
+                      <mat-chip class="small-chips mb-0.5"
+                        ><span class="uppercase">{{ phenotypic_feature.value }} </span
+                        ><strong>{{ phenotypic_feature.count }}</strong></mat-chip
+                      >
+                    </li>
+                  }
+                </ul>
+              }
+            </div>
+          </div>
+        </div>
+        <div class="flex grow flex-col gap-y-8">
+          <div class="flex">
+            <div class="me-3 pt-4 text-5xl text-neutral-700">
+              <mat-icon inline>description</mat-icon>
+            </div>
+            <div>
+              <p><strong>File Summary:</strong></p>
+              <p>
+                <strong>{{ studiesSummary().count }}</strong> File{{
+                  studiesSummary().count === 1 ? '' : 's'
```

Comment:
Might be good to put pluralization into a pipe or something. (Must not be done in this PR, just saying).

Reply:
See here for example https://prototyp.digital/blog/pluralization-in-angular-ngplural